create-cache-permanent/README.md

```diff
@@ -1,88 +1,105 @@
-
-```bash
-$ oc get pods -w                                                                                                                                                master ⬆ ✱ ◼
-NAME                 READY     STATUS      RESTARTS   AGE
-...
-datagrid-service-0   0/1       Running     0          10s
-datagrid-service-0   1/1       Running     0          44s
-```
-
-Once the pod is ready, make a put/get invocation to the cache.
-If the cache definition was made permanent, this invocation should be successful:
-
-```bash
-$  mvn fabric8:run -Pget-cache
-...
-[INFO] F8: --- Got cache, put/get returned: sample-value ---
-```
-
-
-## Code
-
-In this section you will learn about what makes a cache creation permanent.
-
-To do that, the code first connects connects to the datagrid service by instantiating a RemoteCacheManager.
-Then, by calling `mvn fabric8:run -Pcreate-cache` a random cache name is generated and using the RemoteCacheManager, the cache is created:
+# Creating a Permanent Cache
+Learn how to create permanent cache instances with Red Hat Data Grid.
+
+A permanent cache can survive between application restarts so you only need to create the cache once. However, data that resides in the cache is not persisted between restarts unless you configure persistent storage.
+
+This repository contains:
+* A set of Java class files that demonstrate how to programmatically create a cache instance and perform `GET` and `PUT` operations on the cache.
+* A `pom.xml` file that defines properties and dependencies for compiling the Java class files into an application.
+
+## Creating a Data Grid Pod
+
+1. Import the Data Grid service template if it is not already available.
+  ```bash
+  $ oc login -u system:admin
+
+  $ oc create \
+    -n openshift \
+    -f https://raw.githubusercontent.com/jboss-container-images/jboss-datagrid-7-openshift-image/f91b94cfd7da4630ca188cd43c26755ecfc99bdd/services/datagrid-service.json
+  ```
+
+2. Instantiate the template with the Data Grid image.
+  ```bash
+  $ oc login -u developer
+
+  $ oc process openshift//datagrid-service \
+    -p IMAGE=docker-registry.engineering.redhat.com/gzamarre/datagrid72-openshift:JDG-2055 \
+    -p NUMBER_OF_INSTANCES=1 \
+    -p APPLICATION_USER=test \
+    -p APPLICATION_USER_PASSWORD=test \
+    | oc create -f -
+  ```
+
+## Creating a Permanent Cache
+
+1. Change to the `create-cache-permanent` directory in this repository.
+  ```bash
+  $ cd create-cache-permanent
+  ```
+
+2. Use the sample application to create a cache named `custom`.
+  ```bash
+  $ mvn fabric8:run -Pcreate-cache
+  ...
+  [INFO] F8: --- Cache 'custom' created ---
+  ```
+
+3. Add an entry to the cache with a `get/put` invocation.
+  ```bash
+  $ mvn fabric8:run -Pget-cache
+  ...
+  [INFO] F8: --- Got cache, put/get returned: sample-value ---
+  ```
+
+## Verifying that the Cache is Permanent
+
+1. Scale the Data Grid stateful set to `0` replicas.
+  ```bash
+  $ oc get sts
```

Comment:
I'd leave `statefulsets` instead of `sts`, it's better for the reader.

Reply:
I've changed `sts` back to `statefulsets`. Thanks, Galder.